src/callbacks_step/analysis_surface_integral_2d.jl

```diff
@@ -50,7 +50,7 @@ struct ForceState{RealT <: Real}
     psi::Tuple{RealT, RealT} # Unit vector normal or parallel to freestream
     rhoinf::RealT
     uinf::RealT
-    l_inf::RealT
+    linf::RealT
```

Comment:
Why have we switched from `l_inf` to `linf`? Recall that `linf` is used for $L_\infty$ error.